ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx

```diff
@@ -1,160 +1,136 @@
+                        onBlur={handleBlur}
+                    />
                 </Flex>
-            </FormGroup>
+            </FormLabelGroup>
             <PolicyCategoriesSelectField />
-            <Field name="description">
-                {({ field }) => (
-                    <FormGroup fieldId="policy-description" label="Description">
-                        <TextArea
-                            id={field.name}
-                            name={field.name}
-                            value={field.value}
-                            onChange={(event, _value) => onChange(_value, event)}
-                        />
-                        <FormHelperText>
-                            <HelperText>
-                                <HelperTextItem>Enter details about the policy</HelperTextItem>
-                            </HelperText>
-                        </FormHelperText>
-                    </FormGroup>
-                )}
-            </Field>
-            <Field name="rationale">
-                {({ field }) => (
-                    <FormGroup fieldId="policy-rationale" label="Rationale">
-                        <TextArea
-                            id={field.name}
-                            name={field.name}
-                            value={field.value}
-                            onChange={(event, _value) => onChange(_value, event)}
-                        />
-                        <FormHelperText>
-                            <HelperText>
-                                <HelperTextItem>
-                                    Enter an explanation about why this policy exists
-                                </HelperTextItem>
-                            </HelperText>
-                        </FormHelperText>
-                    </FormGroup>
-                )}
-            </Field>
-            <Field name="remediation">
-                {({ field }) => (
-                    <FormGroup fieldId="policy-guidance" label="Guidance">
-                        <TextArea
-                            id={field.name}
-                            name={field.name}
-                            value={field.value}
-                            onChange={(event, _value) => onChange(_value, event)}
-                        />
-                        <FormHelperText>
-                            <HelperText>
-                                <HelperTextItem>
-                                    Enter steps to resolve the violations of this policy
-                                </HelperTextItem>
-                            </HelperText>
-                        </FormHelperText>
-                    </FormGroup>
-                )}
-            </Field>
+            <FormLabelGroup
+                label="Description"
+                fieldId="description"
+                errors={errors}
+                touched={touched}
+                helperText={'Enter details about the policy'}
```

Comment:
Although **description** would sound redundant, **details** is ambiguous because this step is Policy **details**. Can you ask in the channel or even in forum-acs-docs for a meaningful phrase?